### PR TITLE
Enter button to search for zip code

### PR DESCRIPTION
**Pull request by LuisaFCorrea**
#### What problem is this solving?

We need to make the shipping table appears after the enter button is pressed, avoiding the need to press the calculate shipping button.

We were having trouble linking to the workspace that's why we install that "assert" dependency by npm. The screenshot error is here too.

#### How to test it?

[Workspace](https://enterbutton--acctglobal.myvtex.com/pizza-meio-a-meio/p)

#### Screenshots or example usage:

https://www.loom.com/share/5556ede38f1041feb0da05664f3cf135

image-console

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- Calculate shipping when enter is pressed
+- Assert module from node.js
```
Comment (Klynger) [inline CHANGELOG.md:11]: Why?

File: react/components/ShippingSimulator/index.js [hunk 2]
```diff
@@ -77,7 +83,7 @@ const ShippingSimulator = ({
 
   return (
     <Fragment>
-      <div className={`${styles.shippingContainer} t-small c-on-base`}>
+      <div className={`${styles.shippingContainer} t-small c-on-base`} onKeyPress={keyPress}>
```
Comment (Klynger) [inline react/components/ShippingSimulator/index.js:86]: This event is [deprecated](https://developer.mozilla.org/en-US/docs/Web/API/Document/keypress_event), you can use [onKeyDown](https://developer.mozilla.org/pt-BR/docs/Web/API/Document/keydown_event) instead

Comment (kaisermann) [inline react/components/ShippingSimulator/index.js:86]: A clickable div should be tabbable and semantically a button:

```suggestion
      <div className={`${styles.shippingContainer} t-small c-on-base`} onKeyPress={keyPress} tabindex="0" role="button">
```

File: react/components/ShippingSimulator/index.js [hunk 3]
```diff
@@ -41,6 +41,12 @@ const ShippingSimulator = ({
     setIsValid(postalCode.valid)
   }
 
+  const keyPress = e => {
+    if(e.which == 13){
```
Comment (Klynger) [inline react/components/ShippingSimulator/index.js:45]: This attribute is deprecated, use `e.key` (https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key/Key_Values)

**Comment by klzns** [comment]
Thanks for your pull request! Do you mind allowing us to push to your branch?
https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork
So we can fix some issues.


The issue with the assert is happening because your branch is outdated, if you rebase your branch with the latest master, you can remove the assert dependency.

File: react/components/ShippingSimulator/index.js (hunk 2 above)
Comment (klzns) [inline react/components/ShippingSimulator/index.js:86]:
```suggestion
      <div className={`${styles.shippingContainer} t-small c-on-base`}>
```

File: react/components/ShippingSimulator/index.js [hunk 4]
```diff
@@ -77,7 +83,7 @@ const ShippingSimulator = ({
 
   return (
     <Fragment>
-      <div className={`${styles.shippingContainer} t-small c-on-base`}>
+      <div className={`${styles.shippingContainer} t-small c-on-base`} onKeyPress={keyPress}>
         <AddressRules country={country} shouldUseIOFetching>
           <AddressContainer
             Input={StyleguideInput}
```
Comment (klzns) [inline react/components/ShippingSimulator/index.js:89]: I checked the Address Form app and the [PostalCodeGetter have a `onSubmit` prop.](https://github.com/vtex/address-form/blob/master/demo/src/App.js#L147) 

We may use it like:

Add the prop `onSubmit={handleClick}` on `PostalCodeGetter`

File: react/components/ShippingSimulator/index.js [hunk 5]
```diff
@@ -41,6 +41,12 @@ const ShippingSimulator = ({
...
   }
 
+  const keyPress = e => {
+    if(e.which == 13){
+      handleClick(e);
+    }
+  }
+
   const handleClick = e => {
     e.preventDefault()
```
Comment (klzns) [inline react/components/ShippingSimulator/index.js:51]: The prop onSubmit doesn't have an event, so we should check for event before calling `preventDefault`
```suggestion
    e && e.preventDefault()
```

File: react/components/ShippingSimulator/index.js [hunk 6]
```diff
@@ -41,6 +41,12 @@ const ShippingSimulator = ({
     setIsValid(postalCode.valid)
   }
 
+  const keyPress = e => {
+    if(e.which == 13){
+      handleClick(e);
+    }
+  }
+
```
Comment (klzns) [inline react/components/ShippingSimulator/index.js:49]:
```suggestion
```

File: react/package.json [hunk 7]
```diff
@@ -9,6 +9,7 @@
     "@material-ui/core": "^3.5.1",
     "@vtex/css-handles": "^1.1.3",
     "apollo-client": "^2.5.1",
+    "assert": "^2.0.0",
```
Comment (klzns) [inline react/package.json:12]:
```suggestion
```
May remove this, just update your branch to the latest master.

**Comment by LuisaFCorrea** [comment]
@klzns @Klynger can you review again, please?

**Comment by LuisaFCorrea** [comment]
> There are some lint issues and the branch needs a rebase

@Klynger the error are nor from my commits.

File: react/components/ShippingSimulator/index.js [hunk 8]
```diff
@@ -75,15 +75,15 @@ const ShippingSimulator = ({ intl, skuId, seller, country, loaderStyles }) => {
...
+      <div className={`${styles.shippingContainer} t-small c-on-base`} >
         <AddressRules country={country} shouldUseIOFetching>
           <AddressContainer
             Input={StyleguideInput}
             address={address}
             onChangeAddress={handleAddressChange}
             autoCompletePostalCode
           >
-            <PostalCodeGetter />
+            <PostalCodeGetter onSubmit={handleClick}/>
```
Comment (Klynger) [inline react/components/ShippingSimulator/index.js:86]: Here

File: react/components/ShippingSimulator/index.js [hunk 9]
```diff
@@ -75,15 +75,15 @@ const ShippingSimulator = ({ intl, skuId, seller, country, loaderStyles }) => {
 
   return (
     <Fragment>
-      <div className={`${styles.shippingContainer} t-small c-on-base`}>
+      <div className={`${styles.shippingContainer} t-small c-on-base`} >
```
Comment (Klynger) [inline react/components/ShippingSimulator/index.js:78]: Here

**Comment by Klynger** [comment]
> > There are some lint issues and the branch needs a rebase
> 
> @Klynger the error are nor from my commits.

You just added a couple of spaces in some elements and these are errors, so when there are these errors the bot shows some other errors or warnings too, but you just have to fix the ones that I commented now, here is how to do it:

1. Run `yarn` in the root of the app
2. Run `yarn lint --fix`

It will run some checks when you try to commit too, but it should pass

**Comment by LuisaFCorrea** [comment]
> > > There are some lint issues and the branch needs a rebase
> > 
> > 
> > @Klynger the error are nor from my commits.
> 
> You just added a couple of spaces in some elements and these are errors, so when there are these errors the bot shows some other errors or warnings too, but you just have to fix the ones that I commented now, here is how to do it:
> 
> 1. Run `yarn` in the root of the app
> 2. Run `yarn lint --fix`
> 
> It will run some checks when you try to commit too, but it should pass

@Klynger Thank you, I understand now. 

Changes made.

**Comment by Klynger** [comment]
@allcontributors add @LuisaFCorrea for code

**Pull request merged**